### ISSUE TITLE
Can this software be used to calculate the saturated vapor pressure of a single molecule at different temperatures

### ISSUE THREAD

**Issue by zifengdexiatian** [TUHH-TVT/openCOSMO-RS_py#5]
I looked up the relevant data and found that the saturation vapor pressure can be calculated using the cosmo-rs model, but the method given in other literature is to calculate the chemical potential of the liquid and gas phase first, and then calculate the vapor pressure from the chemical potential. However, I found that the example in the software is given to calculate the activity coefficient, so I wonder if it is possible to use the software to calculate the chemical potential, or if there is a method to calculate the vapor pressure directly from the activity coefficient. Thank you very much.

**Comment by simonmb** [comment]
TL;DR
**unfortunately not**

COSMO-RS as an activity coefficient model cannot be directly used to calculate vapor pressures. You can calculate solvation energies and with this you can estimate vapor pressures. However, the work on solvation energies with openCOSMO-RS is not published yet, we are currently working on it. If you are interested in vapor pressures there exist other estimation methods if experimental data is not available. You may find some experimental data [here](https://github.com/CalebBell/thermo) and [here](https://www.chemeo.com).

**Comment by zifengdexiatian** [comment]
Thank you very much for your reply and suggestion
